### ISSUE TITLE
indexer errors

### ISSUE THREAD

**Issue by poplolnman** [pymedusa/Medusa#4712]
**Describe the bug**

basically whenever i set an episode to wanted, these errors keep showing up.

**To Reproduce**

**Expected behavior**

to be able to use those indexers

**Screenshots**

**Medusa (please complete the following information):**
 - OS: [e.g. ubuntu 14.04 x64] windows 10
 - Branch: [e.g. master, develop] master
 - Commit: [e.g. c65ca2] latest

**Logs:**
2018-07-21 16:59:22 WARNING  SEARCHQUEUE-FORCED-11538 :: [NZBGeek] :: [e7c4ff3] Invalid api key. Check your settings
2018-07-21 16:58:57 WARNING  SEARCHQUEUE-FORCED-11538 :: [DOGnzb] :: [e7c4ff3] Invalid api key. Check your settings
2018-07-21 16:58:40 WARNING  SEARCHQUEUE-FORCED-11538 :: [Usenet-Crawler] :: [e7c4ff3] Invalid api key. Check your settings
2018-07-21 16:58:29 WARNING  SEARCHQUEUE-FORCED-11538 :: [altHUB] :: [e7c4ff3] Error parsing caps xml for: altHUB
```
<-- Please replace this whole line with your logs -->
```
</details>

**Additional context**

i have re verified the api keys 2-3 times each and i keep getting theser errors...

**Comment by medariox** [comment]
Check that the API keys are saved in your config.ini. You are probably having issues with your config.ini again for whatever reason.

**Comment by poplolnman** [comment]
im not sure why it wouldnt work because this is a fresh install that i did just yesterday.  when i initially checked the file, the api keys were not there; so i put them in manually. i did another search and still got the same result. however i did check the config and all of the keys were there...

**Comment by OmgImAlexis** [comment]
Did you add the api keys between shutdown or while it was running? If you didn't shutdown then Medusa would have overridden the `config.ini`, we don't autoload new changes.

**Comment by medariox** [comment]
You shouldn't add anything manually to the `config.ini`. Use the appropriate fields from the GUI.

**Comment by poplolnman** [comment]
so i have tried adding the api keys via the GUI again and still nothing. everytime i add it and then save, i go back to it and the api key field is always empty.

**Comment by p0psicles** [comment]
So when you add through gui. Check config.ini.

**Comment by poplolnman** [comment]
i added the api keys via the gui and checked the config.ini, but they are not showing up in it...

**Comment by sharkykh** [comment]
The passwords / API keys in `config.ini` are encrypted.

**Comment by poplolnman** [comment]
not for me, i can see all of them. is there any way i can add these 3 indexers? 2 of them are my main...

**Comment by p0psicles** [comment]
Can you double check the access rights to the config. Something strange is going wrong with that file.

If you change a config setting. Does the date of the file change? Are other settings changed?

**Comment by medariox** [comment]
This isn't your first Medusa install, is it? Considering you are the only one reporting this issue, there must be something different about this new setup, probably something related to permissions or rights, as @p0psicles already said. Maybe you are using an old browser?

**Comment by poplolnman** [comment]
its windows, so it shouldnt be a problem...
i tried to set new settings and date and time did change, but while the setting also changed for a split second in the config; it changed back almost instantly.

**Comment by poplolnman** [comment]
no this isnt my first install and i did change browsers to check, but still nothing. however this is the first time i am using the windows installer. i did try a fresh install directory from source with the same config and backup, and that had the same problem...

**Comment by poplolnman** [comment]
Is there nothing that i can do about this or is there somewhere i could at least go for help?

**Comment by Amward12** [comment]
I had this same exact problem. I couldn't add the API keys in the GUI so i tried adding it manually in the config.ini. For whatever reason it wouldn't save them, kept erasing them. I somehow managed to get NZBGeek to take my API key but it wouldn't save my DogNZB API. It said it was saving but it was saving "" instead of "5555jhfh5h5jh5". So i right clicked to inspect the HTML element and i noticed whatever you put in as your API key value shows up blank

<input type="password" id="dognzb_hash" value="" newznab_name="dognzb_hash" class="newznab_api_key form-control input-sm input350">

I manually added my API key in value="Add API Key Here" hit enter then scrolled to the bottom of the page and hit save and that did the trick. I think something is broken in that HTML element. But this worked for me. Hope this helps

**Comment by Casper042** [comment]
I just had the same problem.

Copy and paste the API Key didn't seem to work.         
for NZBGeek I typed in the key by hand and that seemed to work.
For Usenet Crawler it as too long, so I did the same with Chrome Inspect, modified "value" to be value="apikeyblahblah" and then hit submit.       
API Key now shows the correct number of * in the GUI and the config.ini has been updated as well.

So something about the form validation or how it reacts to pasted in data is broken.

Chrome 71
